<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vera Graphics Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionOpenImage"/>
    <addaction name="actionExport_Bitmap_Tilemap"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPalette">
    <property name="title">
     <string>Palette</string>
    </property>
    <addaction name="actionLoad_gimp_palette"/>
    <addaction name="actionSave_as_Gimp_Palette"/>
    <addaction name="actionReload_Vera_Default"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Palette"/>
    <addaction name="actionExport_Palette_Section_as_Binary"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
    <addaction name="actionHomepage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPalette"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="paletteDock">
   <property name="minimumSize">
    <size>
     <width>205</width>
     <height>200</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Palette</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="paletteDockContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="paletteScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>179</width>
          <height>274</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="configDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Configuration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="imageModBox">
         <property name="title">
          <string>Image Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="iModeBitmapCheck">
            <property name="text">
             <string>Bitmap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="iModeTiled8Check">
            <property name="text">
             <string>Tiled 8x8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="iModeTiled16Check">
            <property name="text">
             <string>Tiled 16x16</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="iModeSpriteCheck">
            <property name="text">
             <string>Sprite</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="spriteWidthBox">
         <property name="title">
          <string>Sprite Width</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="spriteW08Check">
            <property name="text">
             <string>8 Px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spriteW16Check">
            <property name="text">
             <string>16 Px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spriteW32Check">
            <property name="text">
             <string>32 Px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spriteW64Check">
            <property name="text">
             <string>64 Px</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="spriteHeightBox">
         <property name="title">
          <string>Sprite Height</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="spriteH08Check">
            <property name="text">
             <string>8 Px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spriteH16Check">
            <property name="text">
             <string>16 Px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spriteH32Check">
            <property name="text">
             <string>32 Px</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spriteH64Check">
            <property name="text">
             <string>64 Px</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="pixelModeBox">
         <property name="title">
          <string>Pixel Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="pMode1Check">
            <property name="text">
             <string>1 BpP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pMode2Check">
            <property name="text">
             <string>2 BpP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pMode4Check">
            <property name="text">
             <string>4 BpP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pMode8Check">
            <property name="text">
             <string>8 BpP</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="paletteOffsetBox">
         <property name="title">
          <string>Palette Offset</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QRadioButton" name="paletteOffset048Check">
            <property name="text">
             <string>48</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QRadioButton" name="paletteOffset240Check">
            <property name="text">
             <string>240</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="paletteOffset128Check">
            <property name="text">
             <string>128</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="paletteOffset016Check">
            <property name="text">
             <string>16</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QRadioButton" name="paletteOffset208Check">
            <property name="text">
             <string>208</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QRadioButton" name="paletteOffset160Check">
            <property name="text">
             <string>160</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="paletteOffset112Check">
            <property name="text">
             <string>112</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="paletteOffset064Check">
            <property name="text">
             <string>64</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="paletteOffset000Check">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="paletteOffset080Check">
            <property name="text">
             <string>80</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="paletteOffset032Check">
            <property name="text">
             <string>32</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QRadioButton" name="paletteOffset224Check">
            <property name="text">
             <string>224</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QRadioButton" name="paletteOffset176Check">
            <property name="text">
             <string>176</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="paletteOffset192Check">
            <property name="text">
             <string>192</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="paletteOffset144Check">
            <property name="text">
             <string>144</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="paletteOffset096Check">
            <property name="text">
             <string>96</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="zoomSlider">
         <property name="toolTip">
          <string>Zoom level</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tiles:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="NTilesSpin">
         <property name="toolTip">
          <string>Maximum number of tiles</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Transparent Color Index:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="transparentSpin">
         <property name="toolTip">
          <string>Color index for transparent pixels</string>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="separator"/>
   <addaction name="actionShowConverted"/>
   <addaction name="actionShowOriginal"/>
   <addaction name="separator"/>
   <addaction name="actionClosest_RGB"/>
   <addaction name="actionClosest_HSL"/>
   <addaction name="actionClosest_HSV"/>
   <addaction name="separator"/>
   <addaction name="actionExport_Bitmap_Tilemap"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/folder-3.png</normaloff>:/res/folder-3.png</iconset>
   </property>
   <property name="text">
    <string>Open image...</string>
   </property>
   <property name="toolTip">
    <string>Open Image File</string>
   </property>
  </action>
  <action name="actionShowOriginal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
   <property name="toolTip">
    <string>Display the original image</string>
   </property>
  </action>
  <action name="actionShowConverted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Converted</string>
   </property>
   <property name="toolTip">
    <string>Display the converted 8-Bit image</string>
   </property>
  </action>
  <action name="actionClosest_RGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
   <property name="toolTip">
    <string>Look for closest palette color using RGB color space</string>
   </property>
  </action>
  <action name="actionClosest_HSL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HSL</string>
   </property>
   <property name="toolTip">
    <string>Look for closest palette color using Hue/Saturation/Lightness color space</string>
   </property>
  </action>
  <action name="actionLoad_gimp_palette">
   <property name="text">
    <string>Load Gimp Palette...</string>
   </property>
  </action>
  <action name="actionSave_as_Gimp_Palette">
   <property name="text">
    <string>Save as Gimp Palette...</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionClosest_HSV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HSV</string>
   </property>
   <property name="toolTip">
    <string>Look for closest palette color using Hue/Saturation/Brightness color space</string>
   </property>
  </action>
  <action name="actionReload_Vera_Default">
   <property name="text">
    <string>Reload Vera Default</string>
   </property>
   <property name="toolTip">
    <string>Reload Vera Default Palette</string>
   </property>
  </action>
  <action name="actionExport_Bitmap_Tilemap">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/media-floppy-2.png</normaloff>:/res/media-floppy-2.png</iconset>
   </property>
   <property name="text">
    <string>Export Bitmap/Tiles/Sprites...</string>
   </property>
  </action>
  <action name="actionExport_Palette">
   <property name="text">
    <string>Export Palette as Binary...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionExport_Palette_Section_as_Binary">
   <property name="text">
    <string>Export Palette Section as Binary...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
